scheduler/src/main/scala/com.gu.media.scheduler/SchedulerLambda.scala

```diff
@@ -0,0 +1,70 @@
+package com.gu.media.scheduler
+
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+import com.amazonaws.services.lambda.runtime.{Context, RequestHandler}
+import com.gu.contentatom.thrift.atom.media.PrivacyStatus
+import com.gu.media.CapiAccess
+import com.gu.media.lambda.LambdaBase
+import com.gu.media.logging.Logging
+import com.gu.media.youtube.{YouTubeAccess, YouTubeVideos}
+import play.api.libs.json.{JsArray, JsValue}
+
+import scala.annotation.tailrec
+import scala.util.control.NonFatal
+
+class SchedulerLambda extends RequestHandler[Unit, Unit]
+  with LambdaBase
+  with Logging
+  with CapiAccess
+  with YouTubeAccess
+  with YouTubeVideos {
+
+  override def handleRequest(input: Unit, context: Context): Unit = {
+    val now = Instant.now()
+    val oneDayAgo = Instant.now().minus(1, ChronoUnit.DAYS)
+    val assets = getVideosFromScheduledAtoms(1, 100, oneDayAgo, now, Set.empty).filter(isManagedVideo)
+
+    assets.foreach { video =>
+      try {
+        setStatus(video, PrivacyStatus.Public)
```

Comment:
I think this should be the atom's status, as `Unlisted` is a valid option.

Reply:
Looking at this with Retta we realised that the work that this lambda needs to do might be slightly more complex than what it currently does: not only set the privacy of the active asset to public/unlisted, but also to push the updates to the video data yo youtube, as when you schedule it, it is not published. Would it make sense to get the lambda to call the `PublishAtomCommand` endpoint on MAM?
ps I'm on health this week so Retta might pick this up